### PR TITLE
[BACKLOG-38706] WCAG - PIR - Page Zoom: Filter Dialog

### PR DESCRIPTION
**Pull request by jsaraf**
Please review @pentaho/wcag 
To be merged alongside https://github.com/pentaho/pentaho-commons-gwt-modules/pull/994

### CODE REVIEW

File: impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.css [hunk 1]
```diff
@@ -48,6 +132,17 @@ input {
...
+
+:where(.responsive) :is(
+ table:where([role="presentation"]),
+ table:where([role="presentation"]) > *,
+ table:where([role="presentation"]) > * > *,
+ table:where([role="presentation"]) > * > * > *
+) {
+  display: contents;
+  flex: inherit;
+}
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.css:148]: This is too general for this context. Rule should be scoped under `.filter-dialog`

File: impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.css [hunk 2]
```diff
@@ -2,26 +2,110 @@
...
 .filterDialogDetailsContainer {
-  min-height: 250px;
+  min-height: 120px;
 }
 
 .filterDialogMatchContainer {
   height: 50px;
 }
 
+.filterPicklist{
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.css:21]: Formatting is off. Missing space before curly brace. In this one and all other new rules in this file.

File: impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.js [hunk 3]
```diff
@@ -33,6 +33,13 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_TemplatedMixin", "dij
 
   buttons: ['Ok_txt','Cancel_txt'],
 
+  ORIENTATION_LANDSCAPE: 0,
+  ORIENTATION_PORTRAIT: 1,
+  orientation: undefined,
+  _onCancelCallback: undefined,
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.js:39]: What are these properties? Where are they used?

File: impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.js [hunk 4]
```diff
@@ -54,6 +61,7 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_TemplatedMixin", "dij
 
   postCreate: function() {
     this.inherited(arguments);
+    domClass.add(this.popup.domNode,'filter-dialog');
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.js:64]: Missing space after comma.

File: impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.css [hunk 5]
```diff
@@ -2,26 +2,110 @@
   display: none !important;
 }
 
+.responsive.pentaho-common-dialog.filter-dialog .dijitDialogPaneContent{
+  overflow: auto !important;
+}
+
```
Comment (dcleao) [inline impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.css:8]: This stylesheet is shared by responsive and non-responsive themes. Also, the classes assigned to HTML elements were changed. Are the non-responsive themes (crystal and sapphire) still displaying "correctly"?

**Review by dcleao — CHANGES_REQUESTED** [review]

File: impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.js (hunk 3 above)
Comment (jsaraf) [reply impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.js:39]: Added by mistake. Will be removing them.

**Review by dcleao — APPROVED** [review]

**Pull request merged**